packages/account-v2/src/pages/LoginHistory/LoginHistory.tsx

```diff
@@ -1,51 +1,10 @@
 import React from 'react';
-import { Table } from '@deriv-com/ui';
-
-type TRowData = {
-    action?: string;
-    browser?: string;
-    datetime?: string;
-    ipAddress?: string;
-    status?: string;
-};
-const columnOrder = ['datetime', 'action', 'browser', 'ipAddress', 'status'] as const;
-
-const headers = {
-    action: 'Action',
-    browser: 'Browser',
-    datetime: 'Date and Time',
-    ipAddress: 'IP Address',
-    status: 'Status',
-};
-
-const columns = columnOrder.map(key => ({ header: headers[key] }));
+import { useLoginHistory } from '@deriv/api-v2';
+import { Loader } from '@deriv-com/ui';
+import { LoginHistoryTable } from '../../containers';
 
 export const LoginHistory = () => {
-    return (
-        <div className='flex flex-col '>
-            <Table
-                columns={columns}
-                data={[
-                    {
-                        action: 'Login',
-                        browser: 'Chrome v122.0.0.0',
-                        datetime: '2024-03-07 12:04:42 GMT',
-                        ipAddress: '942',
-                        status: 'Successful',
-                    },
-                ]}
-                isFetching={true}
-                // eslint-disable-next-line @typescript-eslint/no-empty-function -- adding empty function until the types are fixed in the Table component
-                loadMoreFunction={() => {}}
-                renderHeader={header => <span>{header}</span>}
-                rowRender={(data: TRowData) => (
-                    <div className='grid grid-flow-col'>
-                        {columnOrder.map(key => (
-                            <div key={key}>{data[key]}</div>
-                        ))}
-                    </div>
-                )}
-            />
-        </div>
-    );
+    const { isLoading, loginHistory } = useLoginHistory(50);
+    if (isLoading) return <Loader />;
+    if (loginHistory?.length) return <LoginHistoryTable loginHistory={loginHistory} />;
```

Comment:
What happens when loginHistory is empty? Do we show some component to notify the user?

Reply:
The value of isSucess will be false if the user has not logged in therefore they will not have access to this section. Ideally LoginHistory will never be empty as logging in ,even for the first time, updates the section and displys the login information 
incase of any error or network issue where API is not called an error is thrown